### PR TITLE
Appraise ruby 2.6.3 and rails 6.0 and drop appraisals for Ruby 2.2.3, Rails 4.0 and 4.1

### PR DESCRIPTION
**Pull request by jimherz**

### CODE REVIEW

**Comment by jimherz** [comment]
[Upgrade biz app to Ruby 2.6.3](https://trello.com/c/FXIMKNel/656-upgrade-biz-app-to-ruby-263)

File: lib/salesforce/column.rb [hunk 1]
```diff
@@ -101,6 +101,8 @@ def self.typecast(type, value)
         when :double
           begin
             BigDecimal(value.to_s)
+          rescue ArgumentError
```
Comment (wpliao1989) [inline lib/salesforce/column.rb]: Minor but I think you can use `bigdecimal/util`'s `to_d`. It's the same implementation.
https://github.com/ruby/ruby/blob/ruby_2_5/ext/bigdecimal/lib/bigdecimal/util.rb#L68

Comment (jimherz) [reply lib/salesforce/column.rb]: Good idea! Thanks

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +9,9 @@ sudo: false
 matrix:
   allow_failures:
     - rvm: ruby-head
+  exclude:
+    - rvm: 2.3.3
```
Comment (wpliao1989) [inline .travis.yml]: I think the commit message should be 2.3?

Comment (jimherz) [reply .travis.yml]: Oy Vey! Good catch :)

**Pull request merged**